connectors/src/connectors/slack/bot.ts

```diff
@@ -392,38 +526,53 @@ async function botAnswerMessage(
           )
         );
       }
-      mentions.push({
+      mention = {
         assistantId: defaultAssistant.sId,
         assistantName: defaultAssistant.name,
-      });
+      };
     }
   }
 
-  const mention = mentions[0];
-  if (mention) {
-    if (!message.includes(":mention")) {
-      // if the message does not contain the mention, we add it as a prefix.
-      message = `:mention[${mention.assistantName}]{sId=${mention.assistantId}} ${message}`;
-    }
-  }
+  const mostPopularAgentConfigurations = [...activeAgentConfigurations]
```

Comment:
Can't we get this from the public API?

Reply:
i did not find any public method to get that - on the other hand we already have the full list of assitant with their usage here